static/css/perdiem.css

```diff
@@ -299,3 +299,22 @@ ul.tabs li {
     margin: 0 auto;
     padding: 20px;
 }
+
```

Comment:
Can these CSS styles be in their own .CSS file?
I'd rather not have common styles in `perdiem.css`